daemon/cmd/daemon_main.go

```diff
@@ -1722,6 +1722,16 @@ func runDaemon() {
 	bootstrapStats.updateMetrics()
 	go d.launchHubble()
 
+	err = option.Config.StoreInFile(option.Config.RunDir)
+	if err != nil {
+		log.WithError(err).Error("Unable to store running configuration")
+	}
+
+	err = option.StoreViperInFile(option.Config.RunDir)
+	if err != nil {
+		log.WithError(err).Error("Unable to store running configuration")
```

Comment:
It would make sense to have a slightly different log msg here than the above so that we can distinguish when one fails.